001-core/privacy.php

```diff
@@ -465,11 +465,12 @@ function delete_old_export_files() {
  * @return string The modified robots.txt content.
  */
 function vip_convenience_domain_robots_txt( $output ) {
-	$host = strtolower( $_SERVER['HTTP_HOST'] ?? '' ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
-	if ( false !== strpos( $host, '.go-vip.co' ) || false !== strpos( $host, '.go-vip.net' ) ) {
+	if ( is_vip_convenience_domain( strtolower( $_SERVER['HTTP_HOST'] ?? '' ) ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
 		$output  = "# Crawling is blocked for go-vip.co and go-vip.net domains\n";
 		$output .= "User-agent: *\n";
 		$output .= "Disallow: /\n";
+	
+		return $output;
```

Comment:
Why is this extra `return` needed now?

Reply:
??? What extra `return`? There's only one in the function

Reply:
Are you sure?

<img width="911" alt="Screenshot 2024-01-05 at 20 38 29" src="https://github.com/Automattic/vip-go-mu-plugins/assets/88371/e820fc20-88e0-47d9-89f9-fa4abd3e6394">